src/libraries/System.Private.Uri/tests/FunctionalTests/UriParameterValidationTest.cs

```diff
@@ -15,31 +15,30 @@ public class UriParameterValidationTest
         public void Uri_MakeRelativeUri_NullParameter_ThrowsArgumentException()
         {
             Assert.Throws<ArgumentNullException>(() =>
-          {
-              Uri baseUri = new Uri("http://localhost/");
-              Uri rel = baseUri.MakeRelativeUri((Uri)null);
-          });
+            {
+                Uri baseUri = new Uri("http://localhost/");
+                _ = baseUri.MakeRelativeUri((Uri)null);
```

Comment:
Presumably it's this second line that's throwing the ArgumentNullException?  We should move the first line to be before the assert and let it get closed over:
```C#
var baseUri = new Uri("http://localhost/");
Assert.Throws<ArgumentNullException>(() => baseUri.MakeRelativeUri((Uri)null));
```

Reply:
Fixed, also made such a change in other places across tests